www/js/widgets/canvas.js

```diff
@@ -1,7 +1,6 @@
 /* global Widget, app, cv */
 
-// 26 ft. 11¼ in. by 52 ft. 5¼ in.
-const s_fieldSize = [629.25, 323.25];
```

Comment:
need to have two (more more) field sizes..  If we accept that we'll access constants in this file, then perhaps we should consolidate - then the mechanism for per-season layouts would allow for this.

2020 field should be:
// 26 ft. 11¼ in. by 52 ft. 5¼ in.
[629.25, 323.25];